style.css

```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400&display=swap');
+
+/* current red in use for theme is #F52319 */
+
+*,
+*::before,
+*::after {
+	box-sizing:border-box;
+	padding: 0;
+	margin: 0;
+	font-family: 'roboto', sans-serif;
+}
+
+*:focus {
+	outline: 0;
+}
+
+/* global rules */
+
+/* nav rules */
+
+nav {
```

Comment:
You're using rems but haven't set your basis html font size, which might be worth doing? Don't know what standard practice is.